app/models/content_item.rb

```diff
@@ -138,6 +138,26 @@ def base_path_without_root
     base_path.sub(%r{^/}, "")
   end
 
+  # This method is temporary. It will be removed when dependency resolution is implemented.
+  # This method, when called on a ContentItem, will return the first Topic ContentItem that is linked
+  # or nil if none exists.
+  def linked_topic
+    topics = self[:links][:topics]
+    return if topics.nil?
+    linked_topic_content_id = topics.first[:content_id]
```

Comment:
Is `first` the correct behaviour? What about content tagged with multiple topics?